client/src/components/ProductDetailPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import axios from 'axios';
+
+class ProductDetailPageComponent extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      productID: this.props.match.params.id,
```

Comment:
I modified the productID in state to pull the id from react-router params instead of using the static file. 
This has the added benefit of making the product pages dynamic but we can always modify this back to a static value for testing.